Bottom: 7e1926276c7be3cf21f102c1e14bd350b1367978
Top:    6ce93a287b86c3d34e0de42a84f69bb3c87d5c53
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:12 +0000

Disable process limits


---

diff --git a/src/polygon.c b/src/polygon.c
index 420772b..fb40faf 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -132,6 +132,7 @@ polygon_init (void)
 
   bw_rotate_circle_seg[0] =  cos_ang;  bw_rotate_circle_seg[1] =  sin_ang;
   bw_rotate_circle_seg[2] = -sin_ang;  bw_rotate_circle_seg[3] =  cos_ang;
+return;
 
   /* DEBUG - AVOID PCB running the system out of memory! */
   getrlimit (RLIMIT_AS, &limit);
